iml-agent/src/action_plugins/stratagem/server.rs

```diff
@@ -370,7 +371,7 @@ pub async fn stream_fidlists(mailbox_files: MailboxFiles) -> Result<(), ImlAgent
                 let xs = xs.into_iter().map(|x| {
                     let x = x?;
 
-                    Ok(format!("{}\n", x).into())
+                    Ok(format!("{{ \"fid\": \"{}\" }}\n", x).into())
                 });
 
                 mailbox_client::send(address.clone(), stream::iter(xs))
```

Comment:
I would have thought this would use the postoffice plugin. Any reason not to use it (besides the extra level of indirection)?

Reply:
The postoffice action plugin creates and destroys mailboxes.

Reply:
I meant creating a new entry in `/etc/iml/postman.conf` and streaming there.

Reply:
The postoffice plugin does handle adding new entries to `/etc/iml/postman.conf`.
Mailbox client handles sending data through the mailbox created via the postoffice plugin.

Reply:
`mailbox::send` currently writes directly to the manager: https://github.com/whamcloud/integrated-manager-for-lustre/blob/1ed25ea2237a089516f3b163ce3df36cc598c1af/iml-agent/src/http_comms/mailbox_client.rs#L22-L27